core/domain/question_domain.py

```diff
@@ -265,17 +270,46 @@ def _convert_state_v31_dict_to_v32_dict(cls, question_state_dict):
 
         return question_state_dict
 
+    @classmethod
+    def _convert_state_v32_dict_to_v33_dict(
+            cls, question_id, question_state_dict):
+        """Converts from version 32 to 33. Version 33 adds
+        dimensions to images in the oppia-noninteractive-image tags
+        located inside tabs and collapsible blocks.
+
+        Args:
+            question_id: str. Question id.
+            question_state_dict: dict. A dict where each key-value pair
+                represents respectively, a state name and a dict used to
+                initalize a State domain object.
+
+        Returns:
+            dict. The converted question_state_dict.
+        """
+        file_system_class = fs_services.get_entity_file_system_class()
+        exploration_fs = fs_domain.AbstractFileSystem(file_system_class(
+            feconf.ENTITY_TYPE_QUESTION, question_id))
+        add_dimensions_to_image_tags = functools.partial(
+            html_validation_service.add_dims_to_img_in_complex_rte,
+            exploration_fs)
+        question_state_dict = state_domain.State.convert_html_fields_in_state( # pylint: disable=line-too-long
```

Comment:
Find a way to avoid the pragma. You can probably introduce a ( after = and break the line there.